stix_shifter_modules/guardium/stix_transmission/delete_connector.py

```diff
@@ -1,26 +1,24 @@
 from stix_shifter_utils.modules.base.stix_transmission.base_delete_connector import BaseDeleteConnector
-import json
 from stix_shifter_utils.utils.error_response import ErrorResponder
-
+from stix_shifter_utils.utils import logger
 
 class DeleteConnector(BaseDeleteConnector):
     def __init__(self, api_client):
         self.api_client = api_client
+        self.logger = logger.set_logger(__name__)
 
     def delete_query_connection(self, search_id):
-        # Not implemented for Guardium
-        responses = json.loads('{"message":"Not Implemented", "status":"Not Implemented","data":{"message":"Cannot delete search - Guardium does not support this request."}}')
-        response_code = 501
-        return_obj = dict()
-        return_obj['success'] = False
-        return_obj['status'] = responses.get('status',"NA")
-        return_obj['progress'] = responses.get('progress',"NA")
-        return_obj['data'] = responses.get('data',"NA")
-        return_obj["search_id"] = search_id
-        # Verify the input
-        if response_code == 202:
-            return_obj['success'] = True
-        else:
-            ErrorResponder.fill_error(return_obj, responses, ['message'])
+        try:
+            response_dict = self.api_client.delete_search(search_id)
+            response_code = response_dict["code"]
 
-        return return_obj
+            # Construct a response object
+            return_obj = dict()
+            if response_code == 200:
+                return_obj['success'] = response_code['success']
```

Comment:
it should be `return_obj['success'] = True`

Reply:
done